prometheus/prometheus.yml

```diff
@@ -37,5 +37,10 @@ scrape_configs:
     static_configs:
       - targets: ["near-sffl-aggregator:9091"]
 
+  - job_name: "sffl-indexer"
```

Comment:
The job should also be added to the operator setup

Reply:
done